OrbitGl/FunctionsDataView.h

```diff
@@ -56,6 +56,8 @@ class FunctionsDataView : public DataView {
   static const std::string kMenuActionDisassembly;
 
  private:
+  static bool ShowSelectedFunctionIcon(const orbit_client_protos::FunctionInfo& function);
+  static bool ShowFrameTrackIcon(const orbit_client_protos::FunctionInfo& function);
```

Comment:
"show" here to me sounds more like performing the actual action. Personally I would use "should show".

Reply:
Yes, agree. Fixed.